client/src/modules/stock/movements/templates/action.cell.html

```diff
@@ -18,6 +18,12 @@
       </a>
     </li>
 
+    <li ng-if="row.entity.stock_requisition_uuid">
+      <a data-method="view-requisition" href ui-sref="stockRequisition({ filters : [{ key : 'reference', value : row.entity.document_requisition, displayValue: displayValue}]})">
```

Comment:
We should probably put "cacheable: false" here.

Reply:
Also, we might want to add the `period` filter with a value 'allTime' to ensure it shows all time.

Reply:
Also, we should be filtering on the UUID of the requisition, not on the text, I think.